config/istio/update-job/job.yaml

```diff
@@ -0,0 +1,92 @@
+#@ load("@ytt:overlay", "overlay")
+
+#@ load("@ytt:data", "data")
+
+---
+# Service account the client will use to reset the statefulset
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: restart-workloads
+  namespace: cf-workloads
```

Comment:
Similar to the above comment. `cf-workloads` should be a configurable setting.

Reply:
Solved in https://github.com/cloudfoundry/cf-k8s-networking/commit/d4aff7abb82aafd9c8ff3640c0b94e4814527a4e